ci-operator/config/shiftstack/shiftstack-ci/shiftstack-shiftstack-ci-main.yaml

```diff
@@ -16,11 +16,17 @@ images:
     RUN python -m pip install yq
   from: src
   inputs:
-    installer:
+    openstack-installer:
       paths:
       - destination_dir: .
         source_path: /bin/openshift-install
-  to: openstack-installer
+  to: shiftstack-quota-image
```

Comment:
As discussed in private, the image name could be something like `shiftstack-ci-scripts` to be highlight the fact that it contains scripts from shiftstack/shiftstack-ci